scripts/ai-help-macros.ts

```diff
@@ -253,6 +298,62 @@ export async function updateEmbeddings(
         console.error(context);
       }
     }
+
+    for (const {
+      mdn_url,
+      text,
+      has_embedding,
+      has_embedding_next,
+    } of embeddingUpdates) {
+      try {
+        console.log(`-> [${mdn_url}] Adding embeddings...`);
+
+        // Embedding for full document.
+        const update = {};
+
+        if (!has_embedding) {
+          const { total_tokens, embedding } = await createEmbedding(
+            text,
+            EMBEDDING_MODEL
+          );
+          Object.assign(update, {
+            total_tokens,
+            embedding: pgvector.toSql(embedding),
+          });
+        }
+
+        if (!has_embedding_next) {
+          const { embedding } = await createEmbedding(
+            text,
+            EMBEDDING_MODEL_NEXT
+          );
+          Object.assign(update, {
+            embedding_next: pgvector.toSql(embedding),
+          });
+        }
+
+        // Create/update document record.
+        const query = {
+          name: "upsert-doc-embeddings",
+          text: `
+            UPDATE mdn_doc_macro
+            SET ${Object.keys(update)
+              .map((key, index) => `${key} = $${index + 2}`)
+              .join(", ")}
```

Comment:
Let's not generate SQL but rather duplicate here.

Reply:
Okay, fixed in [d0e4e95](https://github.com/mdn/yari/pull/10818/commits/d0e4e95aad79affa87244d3830fc6caf5f767522).